### PR TITLE
CAF-1759: Error Handling Guidance Documentation Update

### PR DESCRIPTION
**Pull request by zaramckeown**

### CODE REVIEW

File: worker-framework/docs/pages/en-us/Architecture.md [hunk 1]
```diff
@@ -82,6 +82,48 @@ It also includes:
...
+ propagate the exception). Workers derived from `AbstractWorker<>` should then
+ call `createFailureResult(...)` perhaps using a sensible enumeration status code
+ for your task.
+
+ You do not need to handle the following, as the framework will handle it:
+
+ - The input wrapper not being parsable
+ - Connections to the queue dropping
+ 
 ### Data Store
```
Comment (glucy) [inline worker-framework/docs/pages/en-us/Architecture.md:144]: Are we not adding in the section of Poison Messages here?

File: worker-framework/docs/pages/en-us/Features.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ banner:
     subtitle: Analyze a Larger Range of Formats
```
Comment (glucy) [inline worker-framework/docs/pages/en-us/Features.md:8]: Has the documentation be re-built and re-published?  
  
This comment applies to all the document site content files.

**Review by glucy — CHANGES_REQUESTED** [review]
Reviewed.  Please answer or fix before merging can be completed.

File: worker-framework/docs/pages/en-us/Features.md (hunk 2 above)
Comment (zaramckeown) [reply worker-framework/docs/pages/en-us/Features.md:8]: I was going to do this once the review had been completed. Does that sound okay?

**Comment by zaramckeown** [comment]
Added Poison Messages to Architecture.md

**Review by glucy — APPROVED** [review]

**Pull request merged**